test/jetstream.js

```diff
@@ -64,6 +64,7 @@ if (typeof phantom == "undefined") {
                     server.close();
                     if (code) throw new Error("JetStream failed!");
                     console.log("JetStream completed successfully.");
+                    process.exit();
```

Comment:
Done in https://github.com/mishoo/UglifyJS2/pull/2086/commits/df1c48e5b61911a0af816697b367936d6699a5b9